pkg/utils/json.go

```diff
@@ -42,3 +46,41 @@ func removeNaN(hosts []result.Host) []result.Host {
 	}
 	return newHosts
 }
+
+func GetMetics(searchDir string, m *[]metrics.Metrics) error {
+	resultFilePath := path.Join(path.Dir(path.Clean(searchDir)), "result.txt")
+
+	bytes, err := ioutil.ReadFile(resultFilePath)
+	if err != nil {
+		return err
+	}
+
+	// any line start with '{' and and with '}'
+	r := regexp.MustCompile(`(?m:^{.*}$)`)
```

Comment:
Is this you trying to do a multiline match?

Reply:
Yes.
Found a cool cool to test regex in go.
https://regex-golang.appspot.com/assets/html/index.html

Reply:
Nice, I guess the question is why not just the `^{".*}$` or `^{"\w+":.*}$`?

Reply:
both (?m:^{".*}$) and (?m:^{"\w+":.*}$) would work too.

https://golang.org/pkg/regexp/syntax/

Eg, 
```
m              multi-line mode: ^ and $ match begin/end line in addition to begin/end text (default false)
...
^              at beginning of text or line (flag m=true)
```

Reply:
Are we ever going to have a multiline JSON?